plugins/plugin-chart-table/src/controlPanel.tsx

```diff
@@ -52,7 +52,7 @@ function getQueryMode(controls: ControlStateMapping): QueryMode {
   if (mode === QueryMode.aggregate || mode === QueryMode.raw) {
     return mode as QueryMode;
   }
-  const rawColumns: QueryFormColumn[] | undefined = controls?.all_columns?.value;
+  const rawColumns = controls?.all_columns?.value as QueryFormColumn[] | undefined;
```

Comment:
Is this because of linting?

Reply:
It's because `value` now has its own type (`JsonValue`), previously it was `any` from `AnyDict`.